src/USER-OMP/pair_reaxc_omp.cpp

```diff
@@ -113,7 +113,7 @@ PairReaxCOMP::~PairReaxCOMP()
   if (setup_flag) {
     reax_list * bonds = lists+BONDS;
     for (int i=0; i<bonds->num_intrs; ++i)
-      sfree(bonds->select.bond_list[i].bo_data.CdboReduction, "CdboReduction");
+      sfree(LAMMPS_NS::Pointers::lmp->error, bonds->select.bond_list[i].bo_data.CdboReduction, "CdboReduction");
```

Comment:
shouldn't you have access to `error` directly, since `PairReaxCOMP` derives from `PairReaxC` which derives from `Pointers`?

Reply:
Yes, you are right. I started with lmp->error which gave me an ambiguous reference error during compilation and this was the first workaround I found.